deployment/ccip/changeset/deploy.go

```diff
@@ -473,11 +491,19 @@ func DeployChainContractsForChains(
 		if existingState.Chains[chainSel].LinkToken == nil || existingState.Chains[chainSel].Weth9 == nil {
 			return fmt.Errorf("fee tokens not found for chain %d", chainSel)
 		}
-		err := deployChainContracts(e, chain, ab, rmnHome)
-		if err != nil {
-			e.Logger.Errorw("Failed to deploy chain contracts", "chain", chainSel, "err", err)
-			return fmt.Errorf("failed to deploy chain contracts for chain %d: %w", chainSel, err)
-		}
+		deployGrp.Go(
+			func() error {
+				err := deployChainContracts(e, chain, ab, rmnHome)
```

Comment:
I'm not certain that rmnHome etc are threadsafe - maybe we come back to this idea in a follow up?

Reply:
This does not deal with any home contract write operation. Just deploys chain contracts in different chains